tests/unit/FilingHistoryList1.spec.ts

```diff
@@ -851,12 +862,111 @@ describe('Filing History List - redirections', () => {
             }
           }
       })))
+
+    post.withArgs('businesses/CP1002587/filings?draft=true')
+      .returns(new Promise(resolve => resolve({
+        data:
+      {
+        filing: {
+          header: {
+            availableOnPaperOnly: false,
+            submitter: 'Sarah Smith',
+            date: '2020-04-28 19:14:45 GMT',
+            effectiveDate: '2020-05-06 19:00:00 GMT',
+            filingId: 145190,
+            name: 'correction',
+            status: 'DRAFT'
+          },
+          business,
+          correction: {
+            correctedFilingId: 145190,
+            correctedFilingType: FilingTypes.SPECIAL_RESOLUTION,
+            correctedFilingDate: '2020-04-28 19:14:45 GMT',
+            comment: null
+          },
+          srCorrection
+        }
+      }
+      })))
   })
 
   afterEach(() => {
     sinon.restore()
   })
 
+  it('redirects to Edit URL when filing an Special Resolution Correction', async () => {
+    // init data
+    configurationStore.setConfiguration({ VUE_APP_BUSINESS_EDIT_URL: 'https://edit.url/' })
+    businessStore.setIdentifier('CP1002587')
+    businessStore.setLegalType(CorpTypeCd.COOP)
+    sessionStorage.setItem('BUSINESS_ID', 'CP1002587')
+    sessionStorage.setItem('CURRENT_ACCOUNT', '{ "id": "2816" }')
+    rootStore.keycloakRoles = ['staff']
+    filingHistoryListStore.setFilings([
+      {
+        availableOnPaperOnly: false,
+        businessIdenfier: 'CP1002587',
+        commentsCount: 0,
+        displayName: 'Special Resolution',
+        effectiveDate: 'Wed, 24 May 2023 18:02:16 GMT',
+        filingId: 145190,
+        filingLink: '/CP1002587/filings/145190',
+        isFutureEffective: false,
+        name: FilingTypes.SPECIAL_RESOLUTION,
+        paymentStatusCode: 'COMPLETED',
+        status: FilingStatus.COMPLETED,
+        submittedDate: 'Wed, 24 May 2023 18:02:16 GMT',
+        submitter: 'Sarah Smith'
+      } as any
+    ])
+
+    const wrapper = mount(FilingHistoryList, { vuetify })
+    const vm = wrapper.vm as any
+    await Vue.nextTick()
+
+    // sanity check
+    expect(vm.getFilings.length).toEqual(1)
+
+    // find and click the drop-down menu button
```

Comment:
Alot of redundant comments here - I find that clutters the code 

What is the comment telling you that the code wouldn't tell you? 

Usually comments I find useful for:

- Code that is not clear and easily understandable, and can’t be rewritten properly
- Everything that we can’t explain with well-written code, such as the decisions we made
- etc.


eg.

// find and click the drop-down menu button

    const menuButton = wrapper.find('.menu-btn')
    expect(menuButton).toBeDefined()
    await menuButton.trigger('click')


feels redundant, but that's my opinion. 

Perhaps redundant:

    // wait for a short delay before checking window.location.assign
    await new Promise(resolve => setTimeout(resolve, 100))


A good one (I think this is good, because you explain the intent, which isn't obvious until you read all of the lines below): 

    // verify redirection
    const startCorrectionBtn = wrapper.find('#dialog-start-button')
    expect(startCorrectionBtn.exists()).toBe(true)
    await startCorrectionBtn.trigger('click')
    await Vue.nextTick()

    const editUrl = 'https://edit.url/CP1002587/correction/?correction-id=145190'
    const accountId = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id
    // wait for a short delay before checking window.location.assign
    await new Promise(resolve => setTimeout(resolve, 100))
    expect(window.location.assign).toHaveBeenCalledTimes(1)
    expect(window.location.assign).toHaveBeenCalledWith(editUrl + '&accountid=' + accountId)


Whenever you write a bunch of redundant comments, you get yourself into issues when the code changes. Because you have to do double the work.. update the comment and update the code.